app/controllers/v1/sessions_controller.rb

```diff
@@ -194,7 +192,8 @@ def request_type
     end
 
     def url_service
-      SAML::URLService.new(saml_settings, session: @session_object, user: current_user, params: params)
+      SAML::URLService.new(saml_settings, session: @session_object, user: current_user,
+                                          params: params, loa3_context: LOA::IDME_LOA3)
```

Comment:
this looks fine, but the name of the param, loa3_conext, might be a little misleading as this only changes the idme behavior and not mhv/dslogon right?